src/main/java/org/zalando/nakadi/domain/EventType.java

```diff
@@ -55,13 +57,17 @@
     @Nullable
     private EventTypeOptions options;
 
+    private Optional<Set<String>> writeScope;
+
+    private Optional<Set<String>> readScope;
```

Comment:
I don't really see much benefit of using Optional here.
I think empty set is good enough to describe "empty" case.